libs/utils/test.py

```diff
@@ -87,6 +87,14 @@ def runExperiments(cls):
         test_env = TestEnv(test_conf=cls._getTestConf())
 
         experiments_conf = cls._getExperimentsConf(test_env)
+
+        if ITERATIONS_FROM_CMDLINE:
```

Comment:
Well, we already have a big warning on console to notify the user about the override.

Moreover, I would not be so negative about considering this an hack... nosetest does not support command line parameters and thus, either we can add support for them or use environment variable. An environment variable is still a legal way to defined parameters for an app... ;-)

Moreover, to be honest, I don't think we will even update this code in the near future... mainly because that's doing exactly what we need and it perfectly works for what we need.

However, if you are more comfortable with this name... let's go with it! ;-)